pkg/support/dump.go

```diff
@@ -222,6 +225,22 @@ func (d *agentDumper) DumpHeapPprof(basedir string) error {
 	return DumpHeapPprof(d.fs, basedir)
 }
 
+func (d *agentDumper) DumpOVSPorts(basedir string) error {
+	portsDesc, err := d.ovsCtlClient.DumpPortsDesc()
+	if err != nil {
+		return fmt.Errorf("error when dumping ports desc: %w", err)
+	}
+	portData := make([]string, len(portsDesc))
+	for idx := range portsDesc {
+		portData[idx] = strings.Join(portsDesc[idx], "\n")
+	}
+	err = afero.WriteFile(d.fs, filepath.Join(basedir, "ovsports"), []byte(strings.Join(portData, "\n")), 0644)
```

Comment:
Does it make sense to make a separate func for line 237 to 241?
Then you can do:
`return d.writeFile("ovsports", portData)`